ci/environment-ci.yml

```diff
@@ -17,5 +17,6 @@ dependencies:
 - types-python-dateutil
 - types-pyyaml
 - types-requests
+- pytest-localftpserver
```

Comment:
I did consider using it, but I'm a bit worried that this is a personal project rather than a reliable library.
It only has a dozen of start and a few forks. Furthermore, it does not support macOS.
Not sure if you use macOS (most of us do at B-Open), but I get a warning and it looks like the issue is not going to be fixed: `https://github.com/oz123/pytest-localftpserver/issues/151#issuecomment-922530735`

Reply:
I'm using macOS and it's running fine for me
If it becomes troublesome we can swap it out later